torch/csrc/jit/custom_operator.h

```diff
@@ -180,6 +180,12 @@ Operator createOperator(
       std::tuple_size<ArgumentTuple>::value;
 
   auto schema = torch::jit::detail::inferAndCheckSchema<Traits>(schemaOrName);
+  Symbol name = Symbol::fromQualString(schema.name());
```

Comment:
I don't like this. I want to move our operator registration over to this API and this guard will prevent it. This will need to be removed in the future commit.